packages/cli-dxns/src/dxos.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package dxos;
+
+import "google/protobuf/any.proto";
+
+message Kube {
+  string name = 1;
+  string url = 2;
+  string version = 3;
+}
+
+message Service {
+  string name = 1;
+  string type = 2;
+  string version = 3;
+
+  // Possibility of inheritance for services?
+  google.protobuf.Any service = 4;
```

Comment:
👍 I would even add this to all other message types

Reply:
Maybe use some generic field name like `extension`

Reply:
Also this should be
```
message DxosAny {
  DxosCid schemaCid = 1;
  string mesageFqn = 2;
  bytes payload = 3;
}
```

Reply:
^^ This allows the any field to be not just from the local schema but from any schema registered on DXNS

Reply:
An observation: `DxosAny` is exactly the same as record entry struct in the substrate pallet, maybe this should just be a cid of another DXOS record?

Reply:
@richburdon ^^